src/renderer/components/reader/Reader.tsx

```diff
@@ -182,15 +180,16 @@ interface ReaderState {
     menuOpen: boolean;
     fullscreen: boolean;
     indexes: any;
-    visibleBookmarkList: Locator[];
+    visibleBookmarkList: any[];
```

Comment:
Instead of the `any` type, could this be `visibleBookmarkList: LocatorExtended[];`?